src/Altinn.Profile/Controllers/OrgContactPointController.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+
+using Altinn.Profile.Core;
+using Altinn.Profile.Core.OrganizationNotificationAddresses;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Altinn.Profile.Controllers
+{
+    /// <summary>
+    /// Controller for organization contact point API endpoints for internal consumption (e.g. Notifications) requiring neither authenticated user token nor access token authorization.
+    /// </summary>
+    [Route("profile/api/v1/organizations/contactpoint")]
```

Comment:
@hggutvik suggested to rename this to match internal naming convention
```suggestion
    [Route("profile/api/v1/organizations/notificationaddresses")]
```

Reply:
I'm inclined to agree.

Reply:
Rename the controller as well then.